admin/src/js/controllers/edit-user.js

```diff
@@ -177,6 +191,9 @@ angular.module('controllers').controller('EditUserCtrl',
     };
 
     var validateContactAndFacility = function() {
+      if (!$scope.editUserModel.place && $scope.editUserModel.contact) {
+        return validateRequired('place', 'Facility');
+      }
```

Comment:
I think it would be better if we include this check only when needed (when role is not selected or role is online). Something like this, for example: 
```
if (!role || !role.offline) {
  return !$scope.editUserModel.contact || validateRequired('place', 'Facility');
}
```

Reply:
Nice catch